src/API/WesternStatesWater.WestDaat.Common/DataContracts/OrganizationLoadAllResponse.cs

```diff
@@ -0,0 +1,6 @@
+namespace WesternStatesWater.WestDaat.Common.DataContracts;
+
+public class OrganizationLoadAllResponse : OrganizationLoadResponseBase
+{
+    public List<OrganizationListDetails> Organizations { get; set; }
```

Comment:
This was my fault - I think we slightly prefer to use arrays on the data contracts (we're not consistent in this solution, but for all future additions let's go w/ array)

```suggestion
    public OrganizationListDetails[] Organizations { get; set; }
```